constants/wallets.js

```diff
@@ -3,5 +3,6 @@ module.exports = {
   NEELAM: 'neelam',
   INITIAL_WALLET: {
     dinero: 1000
-  }
+  },
+  SUPPORTED_CURRENCIES: ['dinero', 'neelam']
```

Comment:
This file needs a bit of restructuring I guess so we can reuse the DINERO and NEELAM described above

`CURRENCY_TYPES` could contain 
```JS
{ 
  DINERO: 'dinero',
  NEELAM: 'neelam'
}
```

Then we can use the values of the object `CURRENCY_TYPES` in `SUPPORTED_CURRENCIES`